.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
Here's a little TIL: I just tried things locally and it looks like since the `lint-staged` package is installed with the dev-dependencies the `npx` command doesn't need to go fetch it again.